src/app/(layout-with-navigation)/(simulation)/fin/_components/GridChart.tsx

```diff
@@ -52,7 +52,7 @@ export default function GridChart() {
   return (
     <>
       <h3 className="mb-2 text-center text-lg md:mb-4 md:text-xl">
-        <Trans>De quoi est faite mon empreinte&nbsp;?</Trans>
+        <Trans>De quoi est faite mon empreinte ?</Trans>
```

Comment:
Il n'y a pas moyen de garder les espaces insécables ?

Reply:
Il me semble que la solution serait d'utiliser la fonction `t` au lieu du composant `Trans` :

```suggestion
        {t('De quoi est faite mon empreinte\u202f?')}
```